src/apps/beefy/positions.ts

```diff
@@ -0,0 +1,125 @@
+import { Address } from 'viem'
+import { erc20Abi } from '../../abis/erc-20'
+import { getClient } from '../../runtime/client'
+import { NetworkId } from '../../types/networkId'
+import { toDecimalNumber, toSerializedDecimalNumber } from '../../types/numbers'
+import {
+  AppTokenPositionDefinition,
+  PositionsHook,
+  TokenDefinition,
+  UnknownAppTokenError,
+} from '../../types/positions'
+import { createBatches } from '../../utils/batcher'
+import { BeefyV2AppMulticallAbi } from './abis/BeefyV2AppMulticallAbi'
+import {
+  BeefyVault,
+  NETWORK_ID_TO_BEEFY_BLOCKCHAIN_ID,
+  getAllBeefyVaults,
+  getBeefyLpsPrices,
+} from './api'
+
+// Fetched addresses from https://github.com/beefyfinance/beefy-v2/blob/main/src/config/config.tsx
+const BEEFY_MULTICALL_ADDRESS: {
+  [networkId: string]: Address | undefined
```

Comment:
nit: can we do this, so TS will complain if we add a new network without updating this?
```suggestion
  [networkId: NetworkId]: Address | undefined
```